examples/tutorials/data/hawc.py

```diff
@@ -289,6 +290,6 @@
 # ----------
 #
 # Now you know how to access and work with HAWC data. All other
-# tutorials and documentation concerning 3D analysis and `~gammapy.datasets.MapDataset`s
-# can be used from this step.
+# tutorials and documentation concerning 3D analysis and `~gammapy.datasets.MapDataset`
+# can be used from this step on.
```

Comment:
You need to reword the sentence, without the 's' it does not make sense

Reply:
Right, if we keep the phrasing as it is I could add the link renaming the shown text like `MapDatasets`:

```
`MapDatasets <gammapy.datasets.MapDataset>`
```
although it could be a bit confusing as you might be expecting a class named `MapDatasets`. I'm fine with it, I just wanted to raise the concern. Opinions?